test/test_make_install_with_bench.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
hmm... a little generic question first: why can't this be a part of the other script `test_make_install.sh`?

Reply:
I think this should be separate tests but could partially use the test_make_install.sh so we don't duplicate the code